lib/commands/file-movement.js

```diff
@@ -151,7 +160,12 @@ async function pushFileToRealDevice (device, remotePath, base64Data) {
       dstPath = pathInContainer;
       log.info(`Parsed bundle identifier '${bundleId}' from '${remotePath}'. ` +
         `Will put the data into '${dstPath}'`);
-      ifuseArgs = ['-u', device.udid, '--container', bundleId, mntRoot];
+      if (isDocuments(remotePath)) {
+        mntRoot = await createMountDirForDocuments(mntRoot);
```

Comment:
I would rather have a separate variable for this root, since we'd still like to delete the top folder in the `finally` block